src/components/ActivityCard/ActivityCard.js

```diff
@@ -1,8 +1,16 @@
+import {MdOutlineBookmarkAdd} from 'react-icons/md';
+import styled from 'styled-components';
+
 import Section from './ActivityCardStyle';
 
 export default function ActivityCard(props) {
   return (
     <Section>
+      <FavDiv>
+        <FavButton onClick={() => props.onAddFavorite(props.id)}>
+          <MdOutlineBookmarkAdd />
+        </FavButton>
+      </FavDiv>
       <h2>{props.name}</h2>
       <h3>{props.location}</h3>
       {/* eslint-disable-next-line */}
```

Comment:
Yes. That's because of the issue we already discussed. UL has shorthand `list-style:none`. Therefore i've set `ul role='list'` (accessibility). eslint doesn't recognize `list-style:none` and marks the given `role` as unnecessary